.gitlab-ci.yml

```diff
@@ -212,10 +212,10 @@ publish-docs:
   <<:                              *docker-env
   variables:
     GIT_DEPTH:                     0
-  only:
-    - master
-    - schedules
-    - tags
+  # only:
+  #   - master
+  #   - schedules
+  #   - tags
```

Comment:
Please delete code that should be removed instead of commenting it.

Reply:
it's commented for testing purposes, shouldn't be removed